docs/user-guide/running-crawl.md

```diff
@@ -0,0 +1,40 @@
+# Modifying Running Crawls
+
+## In-Progress Statuses
+
+A crawl workflow that is in progress can be in one of five possible states:
```

Comment:
```suggestion
A crawl workflow that is in progress can be in one of six possible states:
```

Must have added one at some point :)

Reply:
Do we need to state the number? "A crawl workflow that is in progress can be in one of the following states:" or similar also gets the point across

Reply:
```suggestion
A crawl workflow that is in progress can be in one of the following states:
```

Good point!